R/utils_dataprocess_plots.R

```diff
@@ -153,20 +157,26 @@
     text.size, text.angle, legend.size, dot.size.profile, cumGroupAxis, 
     yaxis.name, lineNameAxis, groupNametemp
 ) {
+    print(input)
+    write.csv(input, file = "your_file_name_summ.csv", row.names = T)
```

Comment:
where will this be printed/saved?

Reply:
Commit [remove csv debug](https://github.com/Vitek-Lab/MSstats/pull/107/commits/aea147a3323b1e7885865d54107f00c2fc716205)

Was written to understand the data table